CMakeLists.txt

```diff
@@ -94,22 +96,42 @@ if (UNIX AND NOT APPLE AND NOT ANDROID AND NOT WEBGL)
 endif()
 
 if (LINUX)
-    if (FILAMENT_SUPPORTS_XCB)
-        add_definitions(-DFILAMENT_SUPPORTS_XCB)
+    if (FILAMENT_SUPPORTS_WAYLAND)
+        add_definitions(-DFILAMENT_SUPPORTS_WAYLAND)
+        set(FILAMENT_SUPPORTS_X11 FALSE)
+    else ()
+        if (FILAMENT_SUPPORTS_XCB)
+            add_definitions(-DFILAMENT_SUPPORTS_XCB)
+        endif()
+
+        if (FILAMENT_SUPPORTS_XLIB)
+            add_definitions(-DFILAMENT_SUPPORTS_XLIB)
+        endif()
+
+        if (FILAMENT_SUPPORTS_XCB OR FILAMENT_SUPORTS_XLIB)
+            add_definitions(-DFILAMENT_SUPPORTS_X11)
+            set(FILAMENT_SUPPORTS_X11 TRUE)
+        endif()
     endif()
 
-    if (FILAMENT_SUPPORTS_XLIB)
-        add_definitions(-DFILAMENT_SUPPORTS_XLIB)
+    get_property(LIB64 GLOBAL PROPERTY FIND_LIBRARY_USE_LIB64_PATHS)
+
+    if ("${LIB64}" STREQUAL "TRUE")
+        set(LIBSUFFIX 64)
+    else()
+        set(LIBSUFFIX "")
     endif()
 endif()
 
 if (ANDROID OR WEBGL OR IOS)
     set(IS_MOBILE_TARGET TRUE)
-endif()
+    message(STATUS "Target Build")
+endif ()
 
 if (NOT ANDROID AND NOT WEBGL AND NOT IOS)
     set(IS_HOST_PLATFORM TRUE)
-endif()
+    message(STATUS "Host Build")
```

Comment:
This message isn't necessary and unrelated to your changes

Reply:
will remove